Add note for Windows Server node pool in tutorial-ingress-controller-…

VijayRod opened the pull request:
…add-on-existing

AbelHu commented on articles/application-gateway/tutorial-ingress-controller-add-on-existing.md:
```diff
@@ -24,6 +24,8 @@ In this tutorial, you learn how to:
 > * Deploy a sample application using AGIC for Ingress on the AKS cluster
 > * Check that the application is reachable through Application Gateway
 
+[!NOTE] When using an AKS cluster with a Windows Server node pool, the recommendation is to use Application Gateway in a different subnet within the same virtual network. [PR#1953](https://github.com/Azure/AKS/issues/1953).
```
Is "recommendation" ok since this may be the only solution for now?

johannespetereit replied:
I find the word "recommendation" misleading - when the only working way is deploying both components in the same vnet, than the wording should reflect that. Everything else will not work.

VijayRod replied:
Updated.

dimadhus committed:
update regions

dimadhus committed:
vnet updates

MarileeTurscak-MSFT committed:
Removed deprecated azure-ad-passport link

v-alje committed:
Merge branch 'master' into release-aoa-2108

v-kents committed:
fix merge issue

v-kents committed:
fix merge issue

v-kents committed:
fix merge issue

v-kents committed:
fix merge issue

v-kents committed:
Merge branch 'master' of https://github.com/v-kents/azure-docs-pr

v-kents committed:
Merge branch 'master' of https://github.com/v-kents/azure-docs-pr

v-kents committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

dimadhus committed:
fix vnet comments

dimadhus committed:
change what is new for vnet and same zone ha

v-alje committed:
Merge branch 'master' into release-aoa-2108

shhazam-ms committed:
Subscriptions

shhazam-ms committed:
manage subscriptions line 28 and 41 removed reference to eg

shhazam-ms committed:
Remove certificates file

bwren committed:
Azure Monitor What's New - July 2021

WilliamDAssafMSFT committed:
20210815 update screenshots for synapse public network access

nnnsss731 committed:
Merge branch 'master' into patch-1

asudbring committed:
fixes and updates

asudbring committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lb-update-portal

memildin committed:
Adding Private Link change to AzD for ACR intro

memildin committed:
Removed unnecessary include file

memildin committed:
Fixed broken links

v-kents committed:
Merge branch 'master' of https://github.com/v-kents/azure-docs-pr

BryanLa committed:
Bump start date

CHILIU-MSFT committed:
Update sdk-features.md

CHILIU-MSFT committed:
Update chat-python.md

mimckitt committed:
Update sizes-general.md

vikancha-MSFT committed:
Update and rename NV-series-migration-guide.md to nv-series-migration-guide.md

vikancha-MSFT committed:
Update and rename NV-series-retirement.md to nv-series-retirement.md

vikancha-MSFT committed:
Update TOC.yml

vikancha-MSFT committed:
Update nv-series-retirement.md

vikancha-MSFT committed:
Update nv-series-migration-guide.md

meenalsri committed:
Update synapse-workspace-understand-what-role-you-need.md

JnHs committed:
link to MG topic

atikmapari committed:
Merge branch 'master' into Broken-link-mlearned

shhazam-ms committed:
Manage subscriptions

v-andreaco committed:
Merge pull request #167232 from vikancha-MSFT/patch-30

Create NV-series retirement starting September 1st, 2021

PRMerger17 committed:
Merge pull request #169226 from JnHs/jh-lh-onboard-link-mg

link to MG topic

PRMerger9 committed:
Merge pull request #168138 from atikmapari/Broken-link-mlearned

Broken link fixed

cherylmc committed:
Update TOC.yml

v-alje committed:
Merge branch 'master' into release-aoa-2108

Heidilohr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into work-avd-mmr

v-kents committed:
fix to links broken by the linkfix script

baanders committed:
Update freshness scores

cherylmc committed:
Update faq.md

v-andreaco committed:
Merge pull request #168413 from shhazam-ms/Fastlane--Cloud-permissions

Fastlane: Manage_Subscriptions

bwren committed:
remove headers

andrewbrownmsft committed:
Adding link to schema

roygara committed:
Removing regional info and email limitation.

TimShererWithAquent committed:
Resolve live publish PR issue

TimShererWithAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 08162021authortag

vhorne committed:
add new content from Vamsi Boya

denrea committed:
Merge pull request #169228 from cherylmc/patch-2

Update TOC.yml

PRMerger14 committed:
Merge pull request #169234 from andrewbrownmsft/andbrown081621

Adding link to schema

hpsin committed:
Minor update on key check frequency

v-andreaco committed:
Merge pull request #169230 from baanders/8-6-freshness-update

ADT: Update date freshness

TimShererWithAquent committed:
Fix github author value.

hpsin committed:
`regards` isn't a word? Who knew?

vhorne committed:
fix links

Heidilohr committed:
Fixing cmdlets.

TimShererWithAquent committed:
Merge pull request #169236 from TimShererWithAquent/08162021authortag

Resolve live publish PR issue

PRMerger14 committed:
Merge pull request #169225 from meenalsri/patch-5

Update synapse-workspace-understand-what-role-you-need.md

bwren committed:
fixes

rmca14 committed:
Merge pull request #169209 from MicrosoftDocs/BryanLa-patch-4

Bump start date

PRMerger10 committed:
Merge pull request #169239 from Heidilohr/work-cmdlet-fixes

Fixing cmdlets

vhorne committed:
fix cloud shell header

msmimart committed:
add Australia

PRMerger15 committed:
Merge pull request #169238 from hpsin/patch-104

Minor update on key check frequency

MicrosoftGuyJFlo committed:
WIP

v-rajagt-zz committed:
Update articles/logic-apps/logic-apps-limits-and-config.md

Co-authored-by: Esther Fan <ecfan@users.noreply.github.com>

bwren committed:
links

PRMerger17 committed:
Merge pull request #169241 from msmimart/mm-aus

[B2C] Add Australia to list in Data residency subsection

PRMerger20 committed:
Merge pull request #167732 from jackykang7777/patch-10

Update STT support list with 2 new locales sw-KE, fa-IR

mimckitt committed:
updates

PRMerger20 committed:
Merge pull request #168780 from v-rajagt/estfan-aug9

link fixed.

djpmsft committed:
Adding Glossary terms

djpmsft committed:
adding operation to data plane and control plane

BryanLa committed:
Bump start date

v-donspencer committed:
Merge pull request #169229 from v-kents/fix-sentinel

fix-sentinel

PRMerger17 committed:
Merge pull request #168741 from billmath/fixes4

updating

ecfan committed:
Add note about no evalation for runtime expressions at design time

MicrosoftGuyJFlo committed:
WIP

aazeez442 committed:
Made requested changes to naming of image files

b-hchen committed:
UAE North and UAE Central are now a supported Azure regional pair for cross-region replication 

Added in: 
### Azure regional pairs

* UAE North and UAE Central

ecfan committed:
Promote considerations section up a level

PRMerger14 committed:
Merge pull request #127535 from MartinJaffer-MSFT/patch-21

Update event-hubs-capture-enable-through-portal.md

BeauBraig committed:
Merge pull request #169243 from MicrosoftDocs/BryanLa-patch-1

Bump start date

b-hchen committed:
Supported Azure regional pairs now include UAE North and UAE Central. and  Norway East and Norway West

Added under "Azure regional pairs" subsection 

* UAE North and UAE Central
* Norway East and Norway West

denrea committed:
Merge pull request #169200 from memildin/asc-melvyn-mde-linux

Adding Private Link change to AzD for ACR intro

BeauBraig committed:
Merge pull request #169244 from djpmsft/purview-updates

Updating Glossary with new terms

denrea committed:
Merge pull request #169159 from bwren/am-whats-new

Azure Monitor What's New - July 2021

dimadhus committed:
remove the .

mimckitt committed:
updates

v-donspencer committed:
Merge pull request #169247 from b-juche/patch-247

UAE North and UAE Central are now a supported Azure regional pair for…

aazeez442 committed:
Merge branch 'aazeez/PerceptUpdate' of https://github.com/aazeez442/azure-docs-pr-edits into aazeez/PerceptUpdate

mimckitt committed:
updates

PRMerger14 committed:
Merge pull request #168068 from atikmapari/Broken-link-bwren

Broken link fixed

MicrosoftGuyJFlo committed:
WIP

dimadhus committed:
new link

v-donspencer committed:
Merge pull request #169235 from roygara/sharedRollout

Removing regional info and email limitation for shared disks

mikefrobbins committed:
Corrected azps file names and added sp error info

cherylmc committed:
Update index.yml

vhorne committed:
visual edits

PRMerger20 committed:
Merge pull request #169252 from mikefrobbins/mr-azps-aks-sp-error-081621

Corrected azps file names and added sp error info

asudbring committed:
update article for portal UI changes

asudbring committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lb-update-portal

WilliamDAssafMSFT committed:
20210816 1236 remove old screenshot

denrea committed:
Merge pull request #169219 from mimckitt/vm-selector-tool-update

Add VM selector tool information

vhorne committed:
more visual inspection

aazeez442 committed:
Delete all-components.jpg

asudbring committed:
fix date and screenhosts

asudbring committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lb-update-portal

v-nisska committed:
freshservice doc

dimadhus committed:
Update articles/mysql/flexible-server/whats-new.md

Co-authored-by: Don Spencer <87099566+v-donspencer@users.noreply.github.com>

PRMerger14 committed:
Merge pull request #163350 from WilliamDAssafMSFT/20210622-merge-caveats

20210622 merge caveats

asudbring committed:
fix date and screenhosts

asudbring committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lb-update-portal

vhorne committed:
more header changes

cherylmc committed:
Update faq.md

andrewbrownmsft committed:
Adding message about expired SAS URLs

dimadhus committed:
Update articles/mysql/flexible-server/concepts-networking-vnet.md

Co-authored-by: Don Spencer <87099566+v-donspencer@users.noreply.github.com>

PRMerger14 committed:
Merge pull request #169258 from andrewbrownmsft/andbrown081621a

Adding message about expired SAS URLs

v-donspencer committed:
Merge pull request #168990 from dimadhus/dimadhus/vnetupdates

Dimadhus/vnetupdates

cherylmc committed:
Update TOC.yml

vhorne committed:
more updates

PRMerger20 committed:
Merge pull request #169246 from ecfan/designer

Azure Logic Apps: Designer doesn't evaluate runtime expressions at design time

cherylmc committed:
Update index.yml

v-andreaco committed:
Merge pull request #167441 from aazeez442/aazeez/PerceptUpdate

Speech DDK docs update

vhorne committed:
more edits

v-alje committed:
Taxonomy Metadata Bulk Update

DerekHerman-MSFT committed:
Update storage-files-configure-p2s-vpn-linux.md

add apt update and unzip 
-------
cc: @roygara
GitHub Issue: https://github.com/MicrosoftDocs/azure-docs/issues/79752

v-albemi committed:
Apply suggestions from code review

DerekHerman-MSFT committed:
Update tutorial-manage-disks.md

add azureuser to ssh command
-------
cc: @roygara
GitHub Issue: https://github.com/MicrosoftDocs/azure-docs/issues/79761

spelluru committed:
added links & samples link at the top

PRMerger15 committed:
Merge pull request #169260 from DerekHerman-MSFT/patch-4

Update storage-files-configure-p2s-vpn-linux.md

v-albemi committed:
Merge pull request #169257 from v-nisska/fs

freshservice doc

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#79106

tfitzmac committed:
typo

v-andreaco committed:
Merge pull request #169256 from asudbring/lb-update-portal

Update portal management article for new Load balancer create UI in portal

iafinder committed:
Update TOC.yml

spelluru committed:
updated the date

iafinder committed:
Merge branch 'master' into patch-11

PRMerger13 committed:
Merge pull request #169264 from tfitzmac/0816move

fixes MicrosoftDocs/azure-docs#79106

v-andreaco committed:
Merge pull request #169253 from cherylmc/patch-3

Update index.yml

PRMerger12 committed:
Merge pull request #169261 from DerekHerman-MSFT/patch-5

Update tutorial-manage-disks.md

kgremban committed:
Separate redirect file for iot-edge

v-andreaco committed:
Merge pull request #169237 from vhorne/fwm-policy-ps

add new content from Vamsi Boya

iafinder committed:
Update n-series-migration.md

asudbring committed:
fix headings

asudbring committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into lb-update-portal

WilliamDAssafMSFT committed:
20210816 rename files under 80 char

v-albemi committed:
Merge pull request #169263 from v-alje/bulk-fix-taxonomy-metadata

Taxonomy Metadata Bulk Update

daviburg committed:
SAP trigger pre-requisites and error

Clarified unicode pre-requisites for SAP trigger.
Documented the SAP error message for misconfiguration.
Standardized T-Code to preferred industry casing.

iafinder committed:
Update n-series-migration.md

PRMerger10 committed:
Merge pull request #169268 from asudbring/lb-update-portal

Heading fix patch

tfitzmac committed:
revise rbac steps for portal

PRMerger6 committed:
Merge pull request #169211 from CHILIU-MSFT/CHILIU-MSFT-python-110beta1

Update ACS Chat Python quickstart for 1.1.0-beta.1 release

v-andreaco committed:
Merge pull request #169262 from spelluru/sbusqslinks0816

added links & samples link at the top

v-ryclap committed:
Update to SSPK FPA Licensees List

Update to SSPK FPA Licensees List

huypub committed:
Merge pull request #169267 from MicrosoftDocs/release-aoa-2108

Release aoa 2108

v-andreaco committed:
Merge pull request #169266 from kgremban/aug16-redirect

Separate redirect file for iot-edge

Ja-Dunn committed:
fix

PRMerger14 committed:
Merge pull request #169271 from tfitzmac/0816portalrbac

revise rbac steps for portal

Ja-Dunn committed:
reviewer fix

b-hchen committed:
Added reminder under Considerations to update AD Connections for changed password 

Added bullet:

* If you change the password of the Active Directory user account that is used in Azure NetApp Files, be sure to update the password configured in the [Active Directory Connections](#create-an-active-directory-connection).

daviburg committed:
Update articles/logic-apps/logic-apps-using-sap-connector.md

Co-authored-by: Esther Fan <ecfan@users.noreply.github.com>

huypub committed:
Merge pull request #169232 from MicrosoftDocs/repo_sync_working_branch

Confirm merge from repo_sync_working_branch to master to sync with https://github.com/MicrosoftDocs/azure-docs (branch master)

v-albemi committed:
Merge pull request #169163 from WilliamDAssafMSFT/20210815-synapse-public-network-access

20210815 update screenshots for synapse public network access

PRMerger17 committed:
Merge pull request #169274 from b-juche/patch-248

Added reminder under Considerations to update AD Connections for chan…

v-albemi committed:
Merge pull request #168927 from MicrosoftGuyJFlo/WHFBDeviceRegistrationMigration

[Azure AD] Devices - Device Registration topic migration

v-andreaco committed:
Merge pull request #169050 from MarileeTurscak-MSFT/patch-704

Removed deprecated azure-ad-passport link

PRMerger18 committed:
Merge pull request #168695 from v-hagamp/cisco-anyconnect-git-issue

Product Backlog Item 1509771: SaaS App Tutorial: Cisco AnyConnect update

v-andreaco committed:
Merge pull request #169030 from bhavana-129/rhombus-systems

Product Backlog Item 1511710: SaaS App Tutorial: Rhombus Systems

daviburg committed:
Update articles/logic-apps/logic-apps-using-sap-connector.md

Co-authored-by: Esther Fan <ecfan@users.noreply.github.com>

daviburg committed:
Update articles/logic-apps/logic-apps-using-sap-connector.md

Co-authored-by: Esther Fan <ecfan@users.noreply.github.com>

daviburg committed:
Update articles/logic-apps/logic-apps-using-sap-connector.md

Co-authored-by: Esther Fan <ecfan@users.noreply.github.com>

daviburg committed:
Update articles/logic-apps/logic-apps-using-sap-connector.md

Co-authored-by: Esther Fan <ecfan@users.noreply.github.com>

bwren committed:
Update monitor-virtual-machine.md


-------
cc: @bwren

daviburg committed:
Update logic-apps-using-sap-connector.md

iafinder committed:
Update n-series-migration.md

v-albemi committed:
Merge pull request #168589 from bhavana-129/tutorial-to-update-108

Product Backlog Item 1508941: SaaS App Tutorial: TutorialToUpdate108

denrea committed:
Merge pull request #168694 from v-hagamp/agiloft-git-issue

Product Backlog Item 1509759: SaaS App Tutorial: Agiloft update

mimckitt committed:
Update n-series-migration.md

denrea committed:
Pencil edit

Indented images in revised procedure to align with preceding list entries.

PageWriter-MSFT committed:
Update index.yml

Updated architecture card

Court72 committed:
Apply pencil edits for blocking issues from PR review

denrea committed:
Pencil edit

Reverted previous pencil edit due to warnings.

PRMerger17 committed:
Merge pull request #169272 from v-ryclap/patch-29

Update to SSPK FPA Licensees List

Court72 committed:
Merge pull request #168697 from bhavana-129/tutorial-to-update-109

Product Backlog Item 1509798: SaaS App Tutorial: TutorialToUpdate109

denrea committed:
Merge pull request #168691 from v-hagamp/knowledeowl-issue

Product Backlog Item 1509736: SaaS App Tutorial: KnowledgeOwl update

mimckitt committed:
updates

Heidilohr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into work-avd-mmr

iafinder committed:
Merge pull request #1 from mimckitt/patch-33

Update n-series-migration.md

AlexandraKemperMS committed:
Fixed wording from github announcement

cjgronlund committed:
brand fix per marketing

ShawnJackson committed:
Update azure-monitor-agent-overview.md

v-donspencer committed:
Apply suggestions from code review

Heidilohr committed:
Fixing broken links.

PRMerger6 committed:
Merge pull request #126362 from dcbrown16/patch-2

Update service-tags-overview.md

v-donspencer committed:
Merge pull request #168857 from bhavana-129/tutorial-to-update-110

Product Backlog Item 1510670: SaaS App Tutorial: TutorialToUpdate110

jborsecnik committed:
Merge pull request #167720 from iafinder/patch-11

Add migration guide for multiple N-series retirement

PRMerger16 committed:
Merge pull request #128098 from dcbrown16/patch-4

Update expressroute-faqs.md

ShawnJackson committed:
Update player-widget.md

denrea committed:
Merge pull request #167738 from AlexandraKemperMS/docsthrottlinglimitupdate

Updating Throttling info for 30,000 request limit

PRMerger7 committed:
Merge pull request #169279 from bwren/patch-32

Update monitor-virtual-machine.md

ShawnJackson committed:
Merge pull request #168539 from paulth1/azure-monitor-agent-migration-and-overview

edit pass: Azure monitor agent migration and overview

jborsecnik committed:
Merge pull request #168509 from Veena11/patch-1

Update apple-sso-plugin.md

cjgronlund committed:
Azure Machine Learning Studio (classic) -> ML Studio (classic)

ShawnJackson committed:
Merge pull request #168911 from Ja-Dunn/player-widget

edit pass: player-widget

mimckitt committed:
formatting updates

Heidilohr committed:
Fixing typos and editorial changes for clarity.

denrea committed:
Merge pull request #169282 from PageWriter-MSFT/patch-10

Update index.yml

v-donspencer committed:
Merge pull request #168868 from bhavana-129/tutorial-to-update-111

Product Backlog Item 1510714: SaaS App Tutorial: TutorialToUpdate111

denrea committed:
Pencil edit

Corrected typo "Anatlyics" > "Analytics " on line 46 of transform-data-using-data-lake-analytics.md.

chanuengg committed:
Alt-text updates

chanuengg committed:
Alt-text updates

denrea committed:
Merge pull request #169285 from cjgronlund/brand-fix

brand fix per marketing

PRMerger15 committed:
Merge pull request #169287 from mimckitt/retirement-updates

formatting updates

PRMerger19 committed:
Merge pull request #169289 from chanuengg/pbi081621

Alt-text updates

v-donspencer committed:
Merge pull request #168511 from Heidilohr/work-avd-mmr

Adding new multimedia redirection article to Azure Virtual Desktop documentation

BryanLa committed:
Bump start-date

Commit:
Added Cosmos DB as service supporting MI

PRMerger17 committed:
Merge pull request #169293 from ThomasWeiss/thweiss-cosmosdb-aad-support

Added Cosmos DB as service supporting MI

tamram committed:
Blob inventory GA (pulls from 165281)

PRMerger13 committed:
Merge pull request #169270 from daviburg/patch-9

SAP trigger pre-requisites and error

PRMerger14 committed:
Merge pull request #141144 from GitaraniSharma-MSFT/patch-7

(AzureCXP) MicrosoftDocs/azure-docs#67706

tamram committed:
fix typo

GitHubber17 committed:
Merge pull request #169290 from MicrosoftDocs/BryanLa-patch-2

Bump start-date

nnnsss731 committed:
Update articles/spring-cloud/quickstart-integrate-azure-database-mysql.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

nnnsss731 committed:
Update articles/spring-cloud/quickstart.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

nnnsss731 committed:
Update articles/spring-cloud/quickstart.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

nnnsss731 committed:
Update articles/spring-cloud/quickstart.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

nnnsss731 committed:
Update articles/spring-cloud/quickstart.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

nnnsss731 committed:
Update articles/spring-cloud/quickstart.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

nnnsss731 committed:
Update articles/spring-cloud/quickstart.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

dearandyxu committed:
update control table

PRMerger18 committed:
Merge pull request #169298 from dearandyxu/master

update control table

nnnsss731 committed:
Update articles/spring-cloud/quickstart.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

nnnsss731 committed:
Update articles/spring-cloud/quickstart.md

Co-authored-by: Karl Erickson <karler@microsoft.com>

PRMerger12 committed:
Merge pull request #164800 from tamram/tamram-0707

add new ZRS regions

VijayRod committed:
Merge branch 'master' into patch-6

PRMerger7 committed:
Merge pull request #169294 from tamram/tamram-165281

Blob inventory GA (pulls from 165281)

GitHubber17 committed:
Merge pull request #168226 from nnnsss731/patch-1

Update quickstart.md

tamram committed:
blob inventory - fix wording around billing

PRMerger7 committed:
Merge pull request #169302 from tamram/tamram21-0816a

blob inventory - fix wording around billing

VijayRod committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into patch-6

VijayRod committed:
Update troubleshooting.md
